File: namsor/python/src/CountryFnLnBatchExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    personal_names_1 = models.FirstLastNameIn(
        id="9a3283bd-4efb-4b7b-906c-e3f3c03ea6a4",
        firstName="Keith",
        lastName="Haring",
    )

    personal_names = [
        personal_names_1,
    ]

    batch_first_last_name_in = models.BatchFirstLastNameIn(
        personalNames=personal_names,
    )

    try:
        response = api.PersonalApi(api_client).country_fn_ln_batch(
            batch_first_last_name_in=batch_first_last_name_in,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling PersonalApi#country_fn_ln_batch: %s\n" % e)

File: namsor/ruby/src/CountryFnLnBatchExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

personal_names_1 = NamsorClient::FirstLastNameIn.new
personal_names_1.id = "9a3283bd-4efb-4b7b-906c-e3f3c03ea6a4"
personal_names_1.first_name = "Keith"
personal_names_1.last_name = "Haring"

personal_names = [
    personal_names_1,
]

batch_first_last_name_in = NamsorClient::BatchFirstLastNameIn.new
batch_first_last_name_in.personal_names = personal_names

begin
    response = NamsorClient::PersonalApi.new.country_fn_ln_batch(
        {
            batch_first_last_name_in: batch_first_last_name_in,
        },
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling PersonalApi#country_fn_ln_batch: #{e}"
end
